src/test/java/seedu/dictionote/logic/parser/ParserUtilTest.java

```diff
@@ -224,23 +223,19 @@ public void parseUiActionOption_validValueWithWhitespace_returnsTrimmedTag() thr
     }
 
     @Test
-    public void parsePosition_invalidInput_throwsParseException() {
-        assertThrows(ParseException.class, () -> ParserUtil.parsePosition("10 a"));
+    public void parsePosition_invalidInput_returnZero() {
+        assertEquals(0, ParserUtil.parsePosition("10 a"));
     }
 
     @Test
-    public void parsePosition_outOfRangeInput_throwsParseException() {
-        assertThrows(ParseException.class, MESSAGE_INVALID_POSITION, ()
-            -> ParserUtil.parsePosition(Long.toString(Integer.MAX_VALUE + 1)));
+    public void parsePosition_outOfRangeInput_returnZero() {
 
-        assertThrows(ParseException.class, MESSAGE_INVALID_POSITION, ()
-            -> ParserUtil.parsePosition(Long.toString(10)));
+        int[] invalidRange = {-1, 0, 10, 11};
```

Comment:
Minor, but do we need to test floats?

Reply:
I think its okay though

Reply:
you cant input float into int.............

Reply:
Okay, nvm lol